index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
+                            <p class="paragraph">If you have an application y ou are interested in developing,
+                                a feature that you need built or   a project that needs 
+                                coding. I’d love to help with it 
+                            </p>
+                        </section>
+                            <section>
+                                  <ul class="form-container">
+                                    <li class="input-name input-email-li">
+                                        <input  class="input-email" type="text" name="name" id="name" maxlength="30" placeholder="safari hamuli" required>
+                                    </li>
+                                    <li class="input-email-li">
+                                        <input class="input-email" type="email" name="email" id="email" placeholder="safari@gmail.com" required>
+                                        
+                                    </li>
+                                    <li class="input-email-li">
+                                        <textarea class="text-area" name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                                        placeholder="Enter your message here" required></textarea>
+                                    </li>
+                                    <li  class="submit-button input-email-li">
+                                        <button class="form-btn" type="submit">
+                                            Get in touch
+                                        </button><br>
+                                        <small></small>
+                                    </li>
+                                </ul>
+                           </section>
```

Comment:
- [ ] Kindly wrap the input fields and the submit button in a `form` tag that has both `method` and `action` attributes 👍
- [ ] Kindly center the form inside the container to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200882866-7e817b7c-3619-42bc-9fa7-91c086ead856.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200882916-b9ba8dd4-5299-48a1-aa1e-e88842445034.png)